src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicPreferLengthCountIsEmptyOverAnyFixer.vb

```diff
@@ -0,0 +1,88 @@
+Imports System.Composition
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.VisualBasic
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.NetCore.Analyzers.Performance
+
+Namespace Microsoft.NetCore.VisualBasic.Analyzers.Performance
+    <ExportCodeFixProvider(LanguageNames.VisualBasic), [Shared]>
+    Public NotInheritable Class BasicPreferLengthCountIsEmptyOverAnyFixer
+        Inherits PreferLengthCountIsEmptyOverAnyFixer
+
+        Protected Overrides Function ReplaceAnyWithIsEmpty(root As SyntaxNode, node As SyntaxNode) As SyntaxNode
+            Dim invocation = TryCast(node, InvocationExpressionSyntax)
```

Comment:
Parentheses aren't required for method calls in VB. In that case, we won't have InvocationExpressionSyntax

Reply:
Done.